extensions/xkcd/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "xkcd",
+  "title": "xkcd",
+  "description": "Browse the xkcd comics and keep track of the ones you've read.",
+  "icon": "icon.png",
+  "author": "maggie",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "xkcd",
```

Comment:
For the command title it should be something like **Explore Comics** or **Search Comics**.